src/Map/index.js

```diff
@@ -1,11 +1,11 @@
-import OlMap from 'ol/Map';
-import View from 'ol/View';
-import {  WKT } from 'ol/format';
-import { Tile as TileLayer} from 'ol/layer';
-import { XYZ as XYZSource } from 'ol/source';
-import { get as getProjection } from 'ol/proj';
+import OlMap from "ol/Map";
+import View from "ol/View";
```

Comment:
```suggestion
import View from "ol/View";
```
这里最好还是使用单引号比较好，如果不是html节点的属性都用单引号